[feat/forward-to-cofacts] add Forward lib that forwards new messages to Cofacts

MrOrz opened the pull request:
This PR will ask user to send message to Cofacts when
- there is no rumor found when using Meiyu's matching mechanism
- the conversation takes place in 1-on-1 chat

![image](https://user-images.githubusercontent.com/108608/95646762-3f9a9b80-0afe-11eb-8fca-d64242fb3efc.png)

Note: This may also affect the behavior of Telegram bot, but it's not tested on Telegram.

MrOrz committed:
[feat/forward-to-cofacts] add Forward lib that forwards new messages to Cofacts

MrOrz committed:
[feat/forward-to-cofacts] emphasize Cofacts is automated in wording

CarolHsu commented on app/lib/forward.rb:
```diff
@@ -0,0 +1,69 @@
+      end
+    end
+
+    def forwardToCofacts message
```
ruby style method name with underscore (not necessary)

```suggestion
    def forward_to_cofacts message
```

CarolHsu commented on app/lib/forward.rb:
```diff
@@ -0,0 +1,69 @@
+  class << self
+    def talk(token, message)
+      initiate_client
+      @client.reply_message(token, forwardToCofacts(message))
```
```suggestion
      @client.reply_message(token, forward_to_cofacts(message))
```

CarolHsu commented on app/controllers/listener/lines_controller.rb:
```diff
@@ -26,14 +26,10 @@ def react(event)
-        else
-          check_rumor
-        end
+      if !group_chat? and about_coronavirus?
```
```suggestion
      if !group_chat? && about_coronavirus?
```
ruby use `&&` for and, and `and` is more like flow controlling (?)
https://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby

CarolHsu replied:
and this update is a awesome one! thanks! 🙏

CarolHsu commented on app/controllers/listener/telegrams_controller.rb:
```diff
@@ -3,14 +3,10 @@ class Listener::TelegramsController < ApplicationController
-      else
-        check_rumor
-      end
+    if private_chat? and about_coronavirus?
```
```suggestion
    if private_chat? && about_coronavirus?
```

CarolHsu commented on app/workers/reply_worker.rb:
```diff
@@ -5,17 +5,20 @@ class ReplyWorker
+    if article
+      reply = ReplyDecorator.new(article["articleReplies"], article["id"]).prettify
+      talk(reply)
+    elsif not is_group_chat
```
```suggestion
    elsif (!is_group_chat && platform == 'line')
```

CarolHsu replied:
to get rid of the risk for not working on telegram.

CarolHsu reviewed (changes requested):
hi @MrOrz feel free to pick the suggestion you feel if good to take, instead of `platform == 'line' is highly recommended, the rest of changes could be up to you. Appreciate all the effort you have contributed!

MrOrz committed:
Apply suggestions from code review

- only invoke `Forward` in LINE platform
- use snake_case instead of camelCase

Co-authored-by: shesee <sheseee@gmail.com>
